WalletWasabi.Backend/Controllers/ChaumianCoinJoinController.cs

```diff
@@ -512,6 +522,16 @@ public async Task<IActionResult> PostOutputAsync([FromQuery, Required] long roun
 				}
 			}
 
+			if (round.GetAlicesBy(request.OutputAddress).ToArray().Length != 0)
+			{
+				return BadRequest($"Cannot register an OutputAddress twice.");
+			}
+
+			if (round.GetBobsBy(request.OutputAddress).ToArray().Length != 0)
```

Comment:
This won't help. See https://github.com/nopara73/WalletWasabi/blob/5f6bc59bfb59814e0eaf439209b36599fe54266d/WalletWasabi/Extensions/NBitcoinExtensions.cs#L188